src/native/_mutable.js

```diff
@@ -224,8 +224,8 @@ const makeMDataInfoObj = (mDataInfo) => {
   } catch (err) {
     throw makeError(errConst.XOR_NAME.code, errConst.XOR_NAME.msg(t.XOR_NAME.size));
   }
-  const type_tag = mDataInfo.type_tag;
-  if(!Number.isInteger(type_tag)) throw makeError(errConst.TYPE_TAG_NAN.code, errConst.TYPE_TAG_NAN.msg); 
+  const typeTag = mDataInfo.typeTag;
```

Comment:
This should probably be `const typeTag = mDataInfo.type_tag;`

Reply:
I think this one stands, as the `mDataInfo` is a js construct from line 108 above,.